app/models.py

```diff
@@ -171,10 +180,12 @@ def fido_enabled(self) -> bool:
 
     profile_picture = db.relationship(File, foreign_keys=[profile_picture_id])
 
-    # Use the "via" format for sender address, i.e. "name@example.com via SimpleLogin"
-    # If False, use the format "Name - name at example.com"
-    use_via_format_for_sender = db.Column(
```

Comment:
If we remove this column without migrating to the new `sender_format`, user who has set their preference here will see their choice "disappeared". I guess we could have 2 choices here:

1. modify the migration script to migrate `use_via_format_for_sender` data to `sender_format`
2. keep `use_via_format_for_sender` but deprecate it by `sender_format`.

In my opinion 1. is cleaner but also more difficult :).